runbot_gitlab/runbot_repo.py

```diff
@@ -146,7 +147,16 @@ def set_gitlab_ci_conf(token, gitlab_url, runbot_domain, repo_id):
 
 class RunbotRepo(models.Model):
     _inherit = "runbot.repo"
-    uses_gitlab = fields.Boolean('Use Gitlab')
+
+    uses_gitlab = fields.Boolean(string='Use Gitlab')
```

Comment:
I have tested this just now and apparently it just capitalizes the first word and the other not, perhaps that file should be updated